Source/automap.h

```diff
@@ -14,10 +14,21 @@
 
 namespace devilution {
 
+enum MapExplorationType : uint8_t {

```

Comment:
Why is this not an enum class?

Reply:
because I'm not going to cast this shit everywhere

Reply:
I don't see any reason to cast this ever though?

Reply:
Saving/Loading, comparing if the value is higher for overwriting etc.